### DISCUSSION TITLE
how to import nuxt i18n localePath?

### DISCUSSION THREAD

**Discussion by and2** [nuxt/nuxt#16574] [Questions]
I'm using the @nuxtjs/i18n module and could not figure out how to import the function `localePath` within the script (this is available in the template). 

Of course, there is `nuxt2Context` via `useNuxtApp()` but this is not ideal as it'll break with Nuxt 3. 

Importing `$18n` from `useNuxtApp()` returns the vue-i18n instance rather than nuxti18n so the function is missing there as well. 

Any tips?

**Comment by antlionguard** [comment]
there's any update for this question?

**Comment by nicecue** [comment]
I don't know whether if I injected $nuxt instance into window or not
but i used localePath like
`window.$nuxt.context.localePath`
